selfdrive/car/ford/interface.py

```diff
@@ -39,6 +39,20 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
 
     if ret.flags & FordFlags.CANFD:
       ret.safetyConfigs[-1].safetyParam |= Panda.FLAG_FORD_CANFD
+    else:
+      # Lock out if the car does not have needed lateral and longitudinal control APIs.
+      # Note that we also check CAN for adaptive cruise, but no known signal for LCA exists
+      pscm_config = next((fw for fw in car_fw if fw.ecu == Ecu.eps and b'\x22\xDE\x01' in fw.request), None)
+      if pscm_config:
+        if len(pscm_config.response) != 24:
+          ret.flags |= FordFlags.LKAS_UNAVAILABLE
+          ret.dashcamOnly = True
+        else:
+          config_tja = pscm_config.response[7]  # Traffic Jam Assist
```

Comment:
is this for the long API or part of the lateral API?

Reply:
I don't know what the difference is between the two features (in testing on my car both had to be enabled for the `LateralMotionControl` commands to work) but when we're looking at the PSCM config it would only be impacting lateral.

Reply:
I believe TJA is the combination of LCA and ACC at low speeds

Reply:
Either way the config for the longitudinal API should be set seperately in the ABS/PCM as-built